src/test/java/marquez/api/JobIntegrationTest.java

```diff
@@ -151,18 +154,52 @@ public void testJobRunAfterUpdateEndToEnd() {
             .put(Entity.json(""));
 
     assertEquals(Response.Status.OK.getStatusCode(), res.getStatus());
+
+    final JobRunResponse getJobRunResponse = getJobRunApiResponse(CREATED_JOB_RUN_UUID);
+    assertThat(getJobRunResponse.getRunState()).isEqualTo(JobRunState.State.COMPLETED.name());
+  }
+
+  @Test
+  public void testJobRunAfterMarkedFailedEndToEnd() {
```

Comment:
is this just to increase test coverage which got missed in the previous PR?

Reply:
Yup!